pages/API_Security_Tools.md

```diff
@@ -0,0 +1,66 @@
+---
+
+title: API Security Tools
+layout: col-sidebar
+author: Matt Tesauro
+contributors:
+tags: api, tools, oss
+permalink: /API_Security_Tools
```

Comment:
Technically this works fine. However for user convenience I'd suggest using all lower instead of mixed case. (Same for the actual file name.)

Reply:
ToLower()'ed the permalink and file name.